stable/insights-agent/templates/trivy/_job.yaml

```diff
@@ -6,8 +6,8 @@ volumes:
   emptyDir: {}
 containers:
 - name: trivy
-  image: quay.io/fairwinds/fw-trivy
```

Comment:
Oops! Good catch. Not sure how we did not see this in the first place.

Reply:
Me too!